### PR TITLE
UI+WebDriver: Add an option to enable autoplay globally

### PR DESCRIPTION
**Pull request by tcl3**
This PR adds a command line option `--enable-autoplay`, which enables multimedia autoplay on all websites. An option has also been added to the Qt settings dialog to toggle this option.

The motivation behind this change was to enable autoplay when running WPT tests, as the `webvtt/` tests require autoplay to be enabled, otherwise a significant number of tests time out.

Running `./Meta/WPT.sh run webvtt` gives the following results:

Before:

```
Ran 320 tests finished in 300.6 seconds.
  • 4 ran as expected. 0 tests skipped.
  • 1 tests failed unexpectedly
  • 295 tests timed out unexpectedly
  • 71 tests had unexpected subtest results
```

After:

```
Ran 320 tests finished in 75.5 seconds.
  • 12 ran as expected. 0 tests skipped.
  • 230 tests failed unexpectedly
  • 58 tests timed out unexpectedly
  • 71 tests had unexpected subtest results
```

### CODE REVIEW

File: Ladybird/AppKit/UI/LadybirdWebView.h [hunk 1]
```diff
@@ -86,6 +86,8 @@
 
 - (void)debugRequest:(ByteString const&)request argument:(ByteString const&)argument;
 
+- (void)setAutoplay:(BOOL)enabled;
```
Comment (trflynn89) [inline Ladybird/AppKit/UI/LadybirdWebView.h:89]: Let's call this `setEnableAutoplay` (here and in the files below) in AppKit to match Qt

Comment (trflynn89) [reply Ladybird/AppKit/UI/LadybirdWebView.h:89]: Doesn't look like there's a checkbox added to enable this in AppKit?

File: Userland/Libraries/LibWebView/Options.h [hunk 2]
```diff
@@ -50,6 +55,7 @@ struct ChromeOptions {
     AllowPopups allow_popups { AllowPopups::No };
     DisableScripting disable_scripting { DisableScripting::No };
     DisableSQLDatabase disable_sql_database { DisableSQLDatabase::No };
+    EnableAutoplay enable_autoplay { EnableAutoplay::No };
```
Comment (trflynn89) [inline Userland/Libraries/LibWebView/Options.h:58]: I think this belongs in `WebContentOptions` - it controls the web view rather than the chrome process

Comment (tcl3) [reply Userland/Libraries/LibWebView/Options.h]: I've moved this. There are other options in `ChromeOptions` that should probably be moved as well if that's what the distinction is supposed to be.

**Pull request merged**